vue/components/ui/molecules/filter/filter.vue

```diff
@@ -185,6 +195,7 @@ export const Filter = {
             start: 0,
             itemsToLoad: true,
             loading: false,
+            autoRefreshing: false,

```

Comment:
`filter` can't have any reference to autorefreshing, everything must be in the mixin